src/index.js

```diff
@@ -18,6 +19,7 @@ function component() {
         const task = new TaskObject(description, id, completed);
         AddDelete.addTask(task);
         Store.addLocalStorage(task);
+        document.location.reload();
```

Comment:
- [ ] Kindly, remove this line of code and fix the error that you were having in another way since it's a really bad UX for the user when creating a task. 🪟

Reply:
Thank you for your feedback. I have sorted it out